app.rb

```diff
@@ -13,8 +13,13 @@ class Uploader < Sinatra::Base
 
     get '/healthcheck' do
       {
+        service_status: 'OK',
```

Comment:
@digitalronin if you'll go ahead and approve this, I'll do the consolidated status in a separate PR.